lib/markdown-pdf.js

```diff
@@ -173,7 +173,8 @@ function makePdf(inputHtml, outputPath){
 function processSrc(src){
   //make a local img src path into a "file:///absolute/path/" if it isn't already
 
-  if(url.parse(src).protocol){
+  var protocolsRegEx = /^(https?|file):$/;
+  if(protocolsRegEx.exec(url.parse(src).protocol) != null){
```

Comment:
Don't know that, thanks! In deed, we don't need to parse url at all, so we could probably use :

```
var protocolsRegEx = /^(https?|file):/;  
if(protocolsRegEx.test(src)){
````